packages/cssnano-preset-advanced/src/index.js

```diff
@@ -21,24 +22,51 @@ const autoprefixer = require('autoprefixer');
  */
 
 /**
+ * @typedef {defaultPreset.AutoprefixerOptions} AutoprefixerOptions
+ * @typedef {defaultPreset.BrowserslistOptions} BrowserslistOptions
  * @typedef {defaultPreset.Options & AdvancedOptions} Options
  */
 
-/** @type {AdvancedOptions} */
-const defaultOpts = {
-  autoprefixer: {
-    add: false,
-  },
-};
+/**
+ * @param {NonNullable<Parameters<typeof advancedPreset>[0]>} opts
+ * @returns {NonNullable<Parameters<typeof advancedPreset>[0]>}
+ */
+function defaultOpts(opts) {
+  const sharedOpts = {
+    overrideBrowserslist: opts.overrideBrowserslist,
+    stats: opts.stats,
+    env: opts.env,
+    path: opts.path,
+  };
+
+  /**
+   * @type {Options}
+   */
+  const defaults = {
+    autoprefixer: {
+      add: false,
+      ...sharedOpts,
+
+      // Skip unsupported Browserslist "my stats" strings etc
+      // https://github.com/browserslist/browserslist/pull/237
+      stats:
+        typeof sharedOpts.stats !== 'string'
+          ? sharedOpts.stats // Autoprefixer supports stats object only
+          : undefined,
+    },
+  };
+
+  return deepmerge(defaults, opts);
```

Comment:
Is it possible to avoid adding a dependency? Since it's just used once, we can probably code a solution for this particular case without needing a generic package.